index.html

```diff
@@ -39,10 +41,12 @@ <h3 class="activity">I am a Full Stack Developer</h2>
     </p>
     </div>
     <div class="social-media">
-      <li> <img src="./images/git-icon.png" class="icon git"></a></li>
-      <li> <img src="./images/linkedin-icon.png" class="icon linkedin"></a></li>
-      <li> <img src="./images/victory-icon.png" class="icon victory"></a></li>
-      <li> <img src="./images/m-icon.png" class="icon gmail"></a></li>
+      
+      <div> <img src="./images/git-icon.png" class="icon git"></a></div>
+      <div> <img src="./images/linkedin-icon.png" class="icon linkedin"></a></div>
+      <div> <img src="./images/victory-icon.png" class="icon victory"></a></div>
+      <div> <img src="./images/m-icon.png" class="icon gmail"></a></div>
```

Comment:
- [ ] As the previous reviewer suggested, Kindly ensure that your list of icons is wrapped in a `<ul>` tag and `<li>` tag per icon; and that each icon has a link (a tag).

e.g:

```
<ul>
        <li>
            <a> icon </a>
        </li>
</ul>
```